src/lua/fiber.lua

```diff
@@ -14,6 +15,34 @@ fiber_clock64(void);
 ]]
 local C = ffi.C
 
+local TIMEOUT_INFINITY = 100 * 365 * 86400
+
+local FIBER_SLICE_DEFAULT_BRIEF = [[
+Sets the default value for the max fiber slice. The old value is infinity
+(no warnings or errors). The new value is {warn = 0.5, err = 1.0}.
+
+https://github.com/tarantool/tarantool/wiki/compat%3Afiber_slice_default
```

Comment:
I pushed #8325. Please, adjust the link.

Reply:
Done

```diff
diff --git a/src/lua/fiber.lua b/src/lua/fiber.lua
index cabc62797cb6..3b86ab783b89 100644
--- a/src/lua/fiber.lua
+++ b/src/lua/fiber.lua
@@ -21,7 +21,7 @@ local FIBER_SLICE_DEFAULT_BRIEF = [[
 Sets the default value for the max fiber slice. The old value is infinity
 (no warnings or errors). The new value is {warn = 0.5, err = 1.0}.
 
-https://github.com/tarantool/tarantool/wiki/compat%3Afiber_slice_default
+https://tarantool.io/compat/fiber_slice_default
 ]]
 
 compat.add_option({
```